src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -241,6 +241,19 @@ private async Task<TestResult> RunTestAsyncInternal(WorkItemInfo workItemInfo, O
                     testResultInfo,
                     commandLineArguments,
                     processResults: ImmutableArray.CreateRange(processResultList));
+
+                string getRspDirectory()
+                {
+                    // There is no artifacts directory on Helix, just use the current directory
+                    if (options.UseHelix)
```

Comment:
RunTestAsyncInternal should never get called with options.UseHelix, so not necessary.